Week1/homework/app_new.js

```diff
@@ -0,0 +1,159 @@
+    },
+    anna_karenina: {
+      title: 'Anna Karenina',
+      author: 'Leo Tolstoy',
+      language: 'English',
+      length: 992,
+      info:
+        'Anna is a beautiful, intelligent woman whose passionate affair with the dashing Count Vronsky leads her to ruin. But her story is also about a search for meaning, and by twinning it with that of Levin, an awkward idealist whose happy marriage and domestic trials form the backdrop for a similar quest, Tolstoy creates a rich and complex masterpiece that has captured the imagination of readers for decades.'
+    },
+    animal_farm: {
+      title: 'Animal Farm',
+      author: 'George Orwell',
+      language: 'English',
+      length: 112,
+      info:
+        'Mr Jones of Manor Farm is so lazy and drunken that one day he forgets to feed his livestock. The ensuing rebellion under the leadership of the pigs Napoleon and Snowball leads to the animals taking over the farm. Vowing to eliminate the terrible inequities of the farmyard, the renamed Animal Farm is organised to benefit all who walk on four legs. But as time passes, the ideals of the rebellion are corrupted, then forgotten. And something new and unexpected emerges. . .'
+    },
+    a_tale_of_two_cities: {
+      title: 'A Tale of Two Cities',
+      author: 'Charles Dickens',
+      language: 'English',
+      length: 544,
+      info:
+        'After eighteen years as a political prisoner in the Bastille the aging Dr Manette is finally released and reunited with his daughter in England. There two very different men, Charles Darnay, an exiled French aristocrat, and Sydney Carton, a disreputable but brilliant English lawyer, become enmeshed through their love for Lucie Manette. From the tranquil lanes of London, they are all drawn against their will to the vengeful, bloodstained streets of Paris at the height of the Reign of Terror and soon fall under the lethal shadow of La Guillotine.'
+    },
+    the_alchemist: {
+      title: 'The Alchemist',
+      author: 'Paulo Coelho',
+      language: 'English',
+      length: 197,
+      info:
+        'Paulo Coelhos masterpiece tells the mystical story of Santiago, an Andalusian shepherd boy who yearns to travel in search of a worldly treasure.His quest will lead him to riches far different—and far more satisfying—than he ever imagined.Santiagos journey teaches us about the essential wisdom of listening to our hearts, of recognizing opportunity and learning to read the omens strewn along lifes path, and, most importantly, to follow our dreams.'
+    }
+  };
+  const BookCovers = {
```

Comment:
Variable, function and parameter names should start with a lowercase letter.

Reply:
I was mixed-up Classes and Objects naming conventions. I realized and fixed that issue thanks to review of @yash-kapila . I've missed out "BookCovers" and I fixed it.